mkosi/__init__.py

```diff
@@ -3867,12 +3867,13 @@ def build_stuff(config: MkosiConfig) -> None:
 
         for p in state.config.output_paths():
             if state.staging.joinpath(p.name).exists():
-                os.rename(str(state.staging / p.name), str(p))
+                shutil.move(str(state.staging / p.name), str(p))
                 if p in (state.config.output, state.config.output_split_kernel):
                     compress_output(state.config, p)
+            if state.config.chown and p.exists(): chown_to_running_user(p)
```

Comment:
We're not hard pressed for vertical space and this is much more readable.
```suggestion
            if state.config.chown and p.exists(): 
                chown_to_running_user(p)
```